Sources/Core/Services/PlaybackObservingService.swift

```diff
@@ -65,14 +70,29 @@ final class ModernAVPlayerPlaybackObservingService: PlaybackObservingService {
                                                   object: nil)
     }
     
+    private func hasReallyReachedEndTime(player: AVPlayer) -> Bool {
+        guard
+            let duration = player.currentItem?.duration.seconds
+            else { return false }
+        
+        let currentTime = player.currentTime().seconds + 1
```

Comment:
Maybe a comment to explain why you have to add 1 second ?

Reply:
yep